Server/Components/Pawn/Script/Script.cpp

```diff
@@ -453,3 +454,46 @@ int AMXAPI amx_Register(AMX* amx, const AMX_NATIVE_INFO* list, int number)
 		amx->flags |= AMX_FLAG_NTVREG;
 	return err;
 }
+
+int AMXAPI amx_Allot(AMX* amx, int cells, cell* amx_addr, cell** phys_addr)
```

Comment:
See the amx_FindPublic comments and make sure these functions aren't being hooked by common plugins too